src/install/resolution.zig

```diff
@@ -191,7 +191,7 @@ pub const Resolution = extern struct {
                 .gitlab => try formatter.resolution.value.gitlab.formatAs("gitlab", formatter.buf, layout, opts, writer),
                 .workspace => try std.fmt.format(writer, "workspace://{s}", .{formatter.resolution.value.workspace.slice(formatter.buf)}),
                 .symlink => try std.fmt.format(writer, "link://{s}", .{formatter.resolution.value.symlink.slice(formatter.buf)}),
-                .single_file_module => try std.fmt.format(writer, "link://{s}", .{formatter.resolution.value.symlink.slice(formatter.buf)}),
+                .single_file_module => try std.fmt.format(writer, "module://{s}", .{formatter.resolution.value.single_file_module.slice(formatter.buf)}),
```

Comment:
this single_file_module thing is completely made up and i don't know if we'll keep it

the idea was

what if you could store http imports in package.json?

Reply:
We can drop this altogether in a separate PR if so desired.